### PR TITLE
Add testing for TF-TRT

### PR DESCRIPTION
**Pull request by samikama**
This pr adds a py_test with a trivial class to contrib/tensorrt. Tagging @aaroey

### CODE REVIEW

**Review by aaroey — CHANGES_REQUESTED** [review]

**Comment by samikama** [comment]
@aaroey It should be done now. Thanks for the review.

**Review by aaroey — APPROVED** [review]

File: tensorflow/contrib/tensorrt/BUILD [hunk 1]
```diff
@@ -272,3 +274,15 @@ tf_cc_test(
         "//tensorflow/core:test_main",
     ],
 )
+
+py_test(
+    name = "tf_trt_integration_test",
+    srcs = ["test/tf_trt_integration_test.py"],
+    main = "test/tf_trt_integration_test.py",
+    srcs_version = "PY2AND3",
```
Comment (aaroey) [inline tensorflow/contrib/tensorrt/BUILD]: Please add
```
tags = [                                                                     
    "manual",                                                                
    "notap",                                                                 
],
```
below this line.

**Review by aaroey — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/tensorrt/BUILD [hunk 2]
```diff
@@ -272,3 +274,15 @@ tf_cc_test(
         "//tensorflow/core:test_main",
     ],
 )
+
+py_test(
```
[the remark this answers is not in the document]
Comment (samikama) [reply tensorflow/contrib/tensorrt/BUILD]: @aaroey  It is done but somehow github can't see it

File: tensorflow/contrib/tensorrt/BUILD (hunk 1 above)
Comment (samikama) [reply tensorflow/contrib/tensorrt/BUILD]: @aaroey  Same here

**Review by aaroey — APPROVED** [review]

**Pull request merged**